src/main/Main.cpp

```diff
@@ -206,24 +207,22 @@ try {
     // ETL uses the balancer to extract data.
     // The server uses the balancer to forward RPCs to a rippled node.
     // The balancer itself publishes to streams (transactions_proposed and accounts_proposed)
-    auto balancer = etl::LoadBalancer::make_LoadBalancer(config, ioc, backend, subscriptions, ledgers);
+    auto balancer = etl::LoadBalancer::make_LoadBalancer(config, ioc, backend, subscriptions->get(), ledgers);
```

Comment:
Can we create a single variable above called `subscriptionManager` and do the `->get()` once?Also, `get()` is a scary name, consider renaming 👍